datasets/nasa-usgs-europa-observations.yaml

```diff
@@ -0,0 +1,39 @@
+Name: NASA / USGS Europa Controlled Observations
+Description: 
+Documentation: https://stac.astrogeology.usgs.gov/docs/data/jupiter/europa/galileo_individual_images/
+Contact: https://answers.usgs.gov/
```

Comment:
contact should probably be a NASA rep per below comment.

Reply:
I left the USGS link here only because NASA is not going to be able to answer questions about these data. Sure, they were NASA collected, but the post processing is entirely USGS led and any data questions should be directed to USGS. 

I suspect we are going to see this consistently with the planetary data we will be pushing. The data are collected by some space agency (NASA, JAXA, ESA) and then processed by some entity (a NASA center, USGS, a university lab, etc.). While that processing is almost always under a NASA grant or contract, the expertise to answer questions and describe what has happened resides with the contract or grant holder.

Thoughts?

Reply:
That makes sense thanks for the clarification!